app/views/users/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide :title, t(".all_users") %>
+<h1><%= t".all_users" %></h1>
+<ul class="users">
+    <%= render @users %>
```

Comment:
> <%= render @users %>
dòng này hoạt động như nào thanh niên? :D

Reply:
comment này chưa rep này :D

Reply:
dạ như là vòng lặp để in ra tất cả user ạ